test/validation/custom.validation.spec.ts

```diff
@@ -65,6 +67,37 @@ describe('checkCeasedDateOnOrAfterStartDate', () => {
   });
 });
 
+describe('tests for isUnableToObtainAllTrustInfo', () => {
+  let mockReq = {} as Request;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+
+    mockReq = {
+      session: {} as Session,
+      headers: {},
+      route: '',
+      method: '',
+      body: {},
+    } as Request;
+  });
+
+  test.each([["Yes", true], ["No", false]])("when trust in review and unable_to_obtain_all_trust_info is set to %s, should return %s", (notAllInfoFlag, expectedResult) => {
+    const appData = { update: { review_trusts: [{ review_status: { in_review: true }, unable_to_obtain_all_trust_info: notAllInfoFlag } as Trust] } };
```

Comment:
also the appData should be spread across several lines for readability

Reply:
Done